src/main/java/com/andreamazzarella/contact_manager/Age.java

```diff
@@ -1,36 +1,33 @@
 package com.andreamazzarella.contact_manager;
 
-import java.util.regex.Pattern;
-
 public class Age {
```

Comment:
What's good about this class is if you wanted a user to enter a date of birth instead of a integer, then the whole of your application won't need to change, they just know they're working with an Age type.

`toInt` may want to be named something a bit more descriptive, maybe `inYears()`?